### PR TITLE
Fetch data

### PR DESCRIPTION
**Pull request by kleekich**
@andrewjohnso7 Please review my code and give me a feedback. Thanks!

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,9 +1,27 @@
 const express = require('express');
-
+const bodyParser = require('body-parser');
+const PriceVolume = require('../database/PriceVolume.js');
 const app = express();
 const PORT = 3002;
 
 app.use(express.static(__dirname+'/../client/dist'));
+// parse application/x-www-form-urlencoded
```
Comment (Sensationull) [inline server/index.js]: Let's remove this?

File: server/index.js [hunk 2]
```diff
@@ -1,9 +1,27 @@
...
+app.use(bodyParser.urlencoded({ extended: false }))
+// parse application/json
+app.use(bodyParser.json())
+
+
+app.get('/api/volumes/symbols/:symbolId', function(req, res){
+  console.log(req.params.symbolId);
```
Comment (Sensationull) [inline server/index.js]: Once you've confirmed that this is what you expect, you can take it out

File: server/index.js [hunk 3]
```diff
@@ -1,9 +1,27 @@
...
+  //res.end(JSON.stringify(res.));
+  PriceVolume.find({_id: req.params.symbolId}, (err, data)=>{
+    if(err) console.log(err);
+    console.log(data);
```
Comment (Sensationull) [inline server/index.js]: Same here as above

**Review by Sensationull — COMMENTED** [review]
I'd just take out your comments and console logs. Maybe keep a log in your Engineering Journal as to which comments are necessary for you to remember and where they are

File: server/index.js (hunk 1 above)
Comment (kleekich) [reply server/index.js]: I agree.

File: server/index.js (hunk 2 above)
Comment (kleekich) [reply server/index.js]: Yes. I will take out all console.logs.

File: server/index.js (hunk 3 above)
Comment (kleekich) [reply server/index.js]: Yes. I will take out all console.logs.

Comment (kleekich) [reply server/index.js]: Thank you, Andrew!

**Comment by kleekich** [comment]
@andrewjohnso7 I made changes you suggested Thanks!

**Pull request merged**